### PR TITLE
Psse Introduction

### PR DESCRIPTION
**Pull request by marqueslanauja**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
No

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Docs update

### CODE REVIEW

**Review by zamarrenolm — APPROVED** [review]

File: pages/documentation/grid/formats/psse.md [hunk 1]
```diff
@@ -2,19 +2,130 @@
...
+0 / END OF OWNER DATA, BEGIN FACTS CONTROL DEVICE DATA
+0 / END OF FACTS CONTROL DEVICE DATA, BEGIN SWITCHED SHUNT DATA
+0 / END OF SWITCHED SHUNT DATA, BEGIN GNE DEVICE DATA
+0 / END OF GNE DEVICE DATA, BEGIN INDUCTION MACHINE DATA
+0 / END OF INDUCTION MACHINE DATA, BEGIN SUBSTATION DATA
+0 / END OF SUBSTATION DATA
+Q
+```
+
+```text
```
Comment (mathbagu) [inline pages/documentation/grid/formats/psse.md:50]: use `json` instead?

File: pages/documentation/index.html [hunk 2]
```diff
@@ -184,17 +184,17 @@ <h5>XIIDM</h5>
         </div>
     </div>
 
-    <!-- PSS/E -->
+    <!-- PSS®E -->
     <div class="col-lg-4 col-sm-6 my-2">
         <div class="media documentation_item">
             <div class="doc-icon align-self-start">
                 <img class="doc-icon" src="/assets/img/crystal/crystal_clear_filesystem_folder_html.png" alt="">
```
Comment (mathbagu) [inline pages/documentation/index.html:191]: I think we can change the icon too?

**Review by mathbagu — CHANGES_REQUESTED** [review]

File: pages/documentation/index.html (hunk 2 above)
Comment (zamarrenolm) [reply pages/documentation/index.html:191]: I am not sure we could use an image or icon directly related to the Siemens product

Comment (zamarrenolm) [reply pages/documentation/index.html]: I've added an image taken from the public PSS®E brochure, respecting colours and font

File: pages/documentation/grid/formats/psse.md (hunk 1 above)
Comment (zamarrenolm) [reply pages/documentation/grid/formats/psse.md:50]: used json for RAWX text example

File: pages/documentation/grid/formats/psse.md [hunk 3]
```diff
@@ -2,19 +2,130 @@
 layout: default
 ---
 
-# PSS/E
+# PSS®E
+[PSS®E software](https://new.siemens.com/global/en/products/energy/energy-automation-and-smart-grid/pss-software/pss-e.html) from Siemens provides analysis functions for power system networks in steady-state and dynamic conditions. PSS®E uses different types of files to exchange data about the network. One of them is the RAW file (power flow data file). A PSS®E RAW file contains a collection of unprocessed data that specifies a bus branch network model for the establishment of a power flow working case.
```
Comment (annetill) [inline pages/documentation/grid/formats/psse.md:6]: We use in the web site Bus/Branch and not bus branch

File: pages/documentation/grid/formats/psse.md [hunk 4]
```diff
@@ -2,19 +2,130 @@
...
+```
 
 ## Import
+The import module reads and converts a PSS®E power flow data file to the PowSyBl grid model. The current implementation supports RAW format for versions 33 and 35 and RAWX format for version 35. The import process is performed in three steps:
+- Read input file.
+- Validate input data.
+- Convert input data into PowSyBl grid model.
 
-<span style="color: red">TODO</span>
+First, input data is obtained by reading and parsing the input file and as result a PS®SE model is created in memory. This model can be viewed as a set of Java classes where each data block of the PSS®E model is associated with a specific Java class that describes all their attributes or data items. Then, some inconsistency checks are performed on this model. If the validation succeeds the PSS®E model is converted to a PowSyBl grid model.
```
Comment (annetill) [inline pages/documentation/grid/formats/psse.md:92]: Fix PSSE here

**Review by annetill — CHANGES_REQUESTED** [review]
Thanks a lot, just some small fixes required.

File: pages/documentation/grid/formats/psse.md (hunk 3 above)
Comment (marqueslanauja) [reply pages/documentation/grid/formats/psse.md]: Done

File: pages/documentation/grid/formats/psse.md (hunk 4 above)
Comment (marqueslanauja) [reply pages/documentation/grid/formats/psse.md]: Done

**Review by annetill — APPROVED** [review]
Thanks José Antonio, it is a great introduction !

**Review by mathbagu — APPROVED** [review]

**Pull request merged**